core/src/main/python/wlsdeploy/tool/create/domain_creator.py

```diff
@@ -161,6 +165,7 @@ def create(self):
         self.__fail_mt_1221_domain_creation()
         self.__create_domain()
         self.__deploy()
+        self.__create_boot_dot_properties()
```

Comment:
This should be optional.  The default WebLogic behavior is to never do boot.properties for production mode.  We should not make this automatic.  But, I think its a good feature to have that is especially useful for the Docker/K8S use case.  Just keep in mind that Docker/K8S is not the ONLY use case for WDT.